commit 36c785f051b21728775c9c4f2621d37d586553d0
Author: Simona Vetter <simona.vetter@ffwll.ch>
Date:   Tue Feb 14 22:37:22 2012 +0100

    drm/i915: merge struct intel_gpio into struct intel_gmbus
    
    When we set up the gpio fallback, we always have a 1:1 relationship
    with an intel_gmbus. Exploit that to store all gpio related data in
    there, too. This is a preparation step to merge the tw i2c adapters
    controlling the same bus into one.
    
    Just mundane code-munging in this patch.
    
    Reviewed-by: Eugeni Dodonov <eugeni.dodonov@intel.com>
    Signed-Off-by: Daniel Vetter <daniel.vetter@ffwll.ch>

diff --git a/drivers/gpu/drm/i915/i915_drv.h b/drivers/gpu/drm/i915/i915_drv.h
index 3cc2694a2b93..63cea5082229 100644
--- a/drivers/gpu/drm/i915/i915_drv.h
+++ b/drivers/gpu/drm/i915/i915_drv.h
@@ -302,6 +302,7 @@ struct intel_gmbus {
 	struct i2c_adapter adapter;
 	struct i2c_adapter *force_bit;
 	u32 reg0;
+	u32 gpio_reg;
 	struct drm_i915_private *dev_priv;
 };